shared/src/commonTest/kotlin/com/mbta/tid/mbta_app/viewModel/MapViewModelTests.kt

```diff
@@ -139,4 +169,154 @@ class MapViewModelTests : KoinTest {
             assertEquals(MapViewModel.State.StopSelected(stop, null), awaitItem())
         }
     }
+
+    @Test
+    fun whenInStopDetailsNotResetToAllRailOnAlertChange() = runTest {
```

Comment:
Ugh, these other tests are flaking in CI. Will try to debug a little, but may just ignore them too 😞

Reply:
Made an attempt at that in https://github.com/mbta/mobile_app/pull/1126/commits/31ef166246a3d2851dc46c09df53f7c7bbdaf39b, I think the tricky part is trying to reset the mock midway in order to verify the effect of only the single call we want to test for each. I opted to ignore them all as of that commit but left in the re-write. 

LMK if that isn't quite what you were envisioning! @boringcactus - also would be interested if you have any thoughts on how to make these molecule tests more stable.

Reply:
Oh maybe because what I care about here is more the side effects from LaunchedEffects I can test using composeTestRule instead and `waitUntil`... I'll give that a very small timebox and otherwise just merge as-is with the tests ignored.